src/main/java/org/apache/maven/shared/utils/io/FileUtils.java

```diff
@@ -573,10 +575,10 @@ private static boolean isValidFile( @Nonnull String file, @Nonnull String... ext
     }
 
     /**
-     * Simple way to make a directory
+     * Simple way to make a directory.
      *
      * @param dir the directory to create
-     * @throws IllegalArgumentException if the dir contains illegal Windows characters under Windows OS.
+     * @throws IllegalArgumentException if the dir contains illegal Windows characters under Windows OS
      * @see #INVALID_CHARACTERS_FOR_WINDOWS_FILE_NAME
      */
     public static void mkdir( @Nonnull String dir )
```

Comment:
Why didn't you deprecate this one?

Reply:
This one has some weird logic for handling file names on Windows I'm not sure is available elsewhere.

Reply:
Won't Files#createDirectory()` properly fail in such a case? If so, I would rather rely on this one.

Reply:
OK